package.json

```diff
@@ -153,8 +153,8 @@
     "@babel/plugin-transform-runtime": "^7.10.4",
     "@babel/preset-react": "^7.10.4",
     "@jackfranklin/test-data-bot": "^1.3.0",
-    "@storybook/addon-viewport": "^6.1.21",
-    "@storybook/react": "^6.1.21",
+    "@storybook/addon-viewport": "^6.2.9",
+    "@storybook/react": "^6.2.9",
```

Comment:
Not sure if this is necessary or not, but Storybook would get stuck in a HMR loop when starting on the previous version. Technically out of scope here but given the minor change (and problem that it fixes for me) I think it's okay.